src/data.js

```diff
@@ -89,4 +89,10 @@ var descriptors = [
   "curses",
   "blessings",
   "melancholy"
-];
\ No newline at end of file
+];
+
+module.exports = {
+  covers: covers,
+  titles: titles,
+  descriptors:descriptors
```

Comment:
Lets add a space between the semicolon  and descriptors. Everything else looks good!

Reply:
fixed